templates/server.hcl.j2

```diff
@@ -5,6 +5,10 @@ server {
         bootstrap_expect = {{ nomad_bootstrap_expect }}
     {%- endif %}
 
+    {% if nomad_use_consul != "" %}
```

Comment:
Don't compare to an empty string. `nomad_use_consul` is a Boolean value.
Test with this instead please:
 `if nomad_use_consul is defined and nomad_use_consul|bool`

Reply:
I have fixed it. Please help me review again, @bbaassssiiee .